bot/exts/help_channels/_cog.py

```diff
@@ -297,6 +282,9 @@ async def init_cog(self) -> None:
         await self.bot.wait_until_guild_available()
 
         log.trace("Initialising the cog.")
+        guild = self.bot.get_guild(constants.Guild.id)
```

Comment:
```suggestion
        self.guild = self.bot.get_guild(constants.Guild.id)
```

Reply:
This wasn't actually needed, since I've noticed we don't actually use self.guild anywhere.

So I've updated other parts of the cog to actually use it. See https://github.com/python-discord/bot/pull/1868/commits/03b79560afec92998c14ee8c9914e106319af6c7